wgpu-hal/src/gles/queue.rs

```diff
@@ -933,25 +940,22 @@ impl crate::Queue<super::Api> for super::Queue {
         command_buffers: &[&super::CommandBuffer],
         signal_fence: Option<(&mut super::Fence, crate::FenceValue)>,
     ) -> Result<(), crate::DeviceError> {
-        self.reset_state();
+        let shared = self.shared.clone();
```

Comment:
nit: generally speaking, `Arc::clone(&xx)` is preferable over the regular `xx.clone()`